packages/game-client/src/App.jsx

```diff
@@ -168,6 +175,16 @@ function App() {
 		directions.current[snakeId] = direction;
 	};
 
+	//gets all the hashes of snakes and returns it as an object :)
+
+	const getSnakeHash = (state) => {
+		const combinedHash = Object.values(state).reduce((result, obj) => {
+		Object.assign(result, obj.hash);
```

Comment:
Add a check here, to make sure that the snakes data has integrity... By integrity, make sure that All the keys are unique, if not throw an error, throw new Error('The snakes data is corrupt, snakes with id 1 and 2 have the same cell')

Reply:
Name the local variables and variables in general properly.